HousingRepairsOnlineApi/UseCases/ChangeRepairRequestUseCase.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Threading.Tasks;
+using Ardalis.GuardClauses;
+using HousingRepairsOnlineApi.Domain;
+using HousingRepairsOnlineApi.Gateways;
+using HousingRepairsOnlineApi.Helpers;
+
+namespace HousingRepairsOnlineApi.UseCases
+{
+    public class ChangeRepairRequestUseCase : IChangeRepairRequestUseCase
+    {
+        private readonly IRepairStorageGateway repairStorageGateway;
+
+        public ChangeRepairRequestUseCase(IRepairStorageGateway repairStorageGateway)
+        {
+            this.repairStorageGateway = repairStorageGateway;
+        }
+
+        public async Task Execute(Repair repair, DateTime startDateTime, DateTime endDateTime)
+        {
+            Guard.Against.Null(repair, nameof(repair));
+            repair.Time.StartDateTime = startDateTime;
+            repair.Time.EndDateTime = endDateTime;
+            await repairStorageGateway.ModifyRepair(repair);
```

Comment:
This saves the repair, but that isn't reflected in the name of the use case.

To apply SRP, only the modifying of the repair time OR saving the modified repair should be the focus of the use case.
Changing the property values of `repair.Time` is causing a side effect and would impact any other referenced instances.